unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/GLTFast/AssetPromise_GLTFast_Instance.cs

```diff
@@ -106,10 +108,14 @@ private IEnumerator LoadingCoroutine(Action OnSuccess, Action<Exception> OnFail)
 
             if (success)
             {
-                if (asset.container == null) { Debug.LogError("this should not happen"); }
+                if (asset.container == null)
```

Comment:
null-checking a `GameObject` could be hazardous, maybe we could take the opportunity to change the condition for `if (!asset.container)`?